### PR TITLE
Feature/tablet

### PR DESCRIPTION
**Pull request by ossset**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -275,28 +284,30 @@ <h2 class="individual__title">
                     <span class="individual__pager-link"></span>
                     <span class="individual__pager-link individual__pager-link--last"></span>
                 </div>
-                <div class="individual__pager-left">
+                <div class="slider-button slider-button--left">
                 </div>
-                <div class="individual__name individual__name--cherry individual__name--3">
+                <div class="individual__slider-container">
+                    <div class="individual__name individual__name--cherry individual__name--design">
```
Comment (uladzislau-kirychenka) [inline index.html]: Please give the element `name` more suitable name like `slider-item`

File: index.html [hunk 2]
```diff
@@ -275,28 +284,30 @@ <h2 class="individual__title">
                     <span class="individual__pager-link"></span>
                     <span class="individual__pager-link individual__pager-link--last"></span>
                 </div>
-                <div class="individual__pager-left">
+                <div class="slider-button slider-button--left">
```
Comment (uladzislau-kirychenka) [inline index.html:289]: It's still an element of `individual` block, change name to `individual__slider-button`

File: index.html [hunk 3]
```diff
@@ -527,7 +538,7 @@ <h2 class="algorithm__title">
             <form action="#" method="POST" class="prefooter__form">
                 <input type="text" placeholder="Ваше имя" class="prefooter__item prefooter__item--input" />
                 <input type="email" placeholder="Email адрес" class="prefooter__item prefooter__item--input" />
-                <textarea name="question" id="question" cols="28" rows="8"
+                <textarea  name="question" id="question" cols="28" rows="8"
```
Comment (uladzislau-kirychenka) [inline index.html]: Extra space

File: css/main.css [hunk 4]
```diff
@@ -38,1357 +40,94 @@ ul {
...
-    content: '1/26';
+    background-color: #f50049;
 
-    font-style: normal;
+    font-size: 13px;
 }
 
-.feedback__link:hover {
-    cursor: pointer;
+.button_small {
```
Comment (uladzislau-kirychenka) [inline css/main.css]: As far as `small` is modifier, rename it to `button--small`

File: css/promo.css [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+    }
+
+    .promo__cons {
+        display: flex;
+        align-items: center;
+
+        padding: 145px 0 0 30px;
+    }
+
+    .promo__item--present,
```
Comment (uladzislau-kirychenka) [inline css/promo.css]: No need to give modifier same styles as element already has

File: css/examples.css [hunk 6]
```diff
@@ -0,0 +1,181 @@
+.examples__card {
+    position: relative;
+
+    width: 280px;
+    height: 120px;
+    margin: 0 auto;
+    margin-bottom: 10px;
```
Comment (uladzislau-kirychenka) [inline css/examples.css]: Use margin shortcut above to set `margin-bottom`

File: css/individual.css [hunk 7]
```diff
@@ -0,0 +1,530 @@
...
+.individual {
+    padding-bottom: 20px;
+
+    background-color: #fbebea;
+}
+
+.individual__title {
+    width: 216px;
+    margin: 0 auto;
+    padding-bottom: 36px;
```
Comment (uladzislau-kirychenka) [inline css/individual.css]: Use padding shortcut

**Review by uladzislau-kirychenka — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (ossset) [reply index.html:289]: This button used in ```about-us``` section as well, do I need to rename it?

File: index.html (hunk 1 above)
Comment (ossset) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (ossset) [reply index.html]: done

File: css/main.css (hunk 4 above)
Comment (ossset) [reply css/main.css]: done

File: css/promo.css (hunk 5 above)
Comment (ossset) [reply css/promo.css]: done

File: css/examples.css (hunk 6 above)
Comment (ossset) [reply css/examples.css]: done

File: css/individual.css (hunk 7 above)
Comment (ossset) [reply css/individual.css]: done

**Review by vlad-kirychenka — APPROVED** [review]

**Review by uladzislau-kirychenka — APPROVED** [review]

**Pull request merged**